src/Agent/NewRelic/Agent/Core/AgentHealth/AgentHealthReporter.cs

```diff
@@ -985,11 +985,11 @@ private void ReportIfAwsAccountIdProvided()
             }
         }
 
-        private void ReportIfAzureFunctionModeIsEnabled()
+        private void ReportIfAzureFunctionModeIsDetected()
         {
-            if (_configuration.AzureFunctionModeEnabled && _configuration.AzureFunctionModeDetected)
+            if (_configuration.AzureFunctionModeDetected)
             {
-                ReportSupportabilityCountMetric(MetricNames.SupportabilityAzureFunctionModeEnabled);
+                ReportSupportabilityCountMetric(MetricNames.SupportabilityAzureFunctionModeEnabled + $"/{(_configuration.AzureFunctionModeEnabled ? "1" : "0")}");
```

Comment:
We should see if there are similar metrics for enabled/disabled things just to see if this naming convention is consistent.

Reply:
Looks like we would typically use `/enabled` or `/disabled` -- I'll update to be more consistent.